global.json

```diff
@@ -1,5 +1,9 @@
 {
   "msbuild-sdks": {
     "Microsoft.Build.Traversal": "1.0.45"
+  },
+  "sdk": {
```

Comment:
@pakrym @heaths @AlexGhiondea do you guys have any issues with us switching to this model for declaring the sdk version?

Reply:
No. And, in fact, I just posted the question last Friday night to our lang channel about doing this very thing because of problems I ran into (VS used a newer version of the SDK than the build agent, which choked on some incidental use of newer lang features).